src/core/client/ui/components/Backdrop/Backdrop.tsx

```diff
@@ -0,0 +1,35 @@
+import cn from "classnames";
+import React, { HTMLAttributes, StatelessComponent } from "react";
+
+import { withStyles } from "talk-ui/hocs";
+
+import styles from "./Backdrop.css";
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  className?: string;
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes: typeof styles;
+  active?: boolean;
+}
+
+const Backdrop: StatelessComponent<Props> = ({
+  classes,
+  active,
+  className,
+  children,
+  ...rest
+}) => {
+  const rootClassName = cn(
+    classes.root,
+    {
+      [classes.active]: Boolean(active),
+    },
+    className
+  );
+  return <div className={rootClassName} {...rest} />;
```

Comment:
Is there a reason there is no `z-index` specification?

Reply:
Yes! This is a general purpose backdrop, you might want to use it for a modal, or maybe you just want to highlight a certain ui element by darken everything except the element. What I mean is, that you don't always apply an z-Index, and the z-Index will depend on where this Backdrop is used in and for what.